src/app/globals.css

```
@@ -25,3 +25,17 @@ body {
     )
     rgb(var(--background-start-rgb));
 }
+
+@layer utilities {
+  .scrollbar::-webkit-scrollbar {
+    width: 2px;
+  }
+ 
+  .scrollbar::-webkit-scrollbar-thumb {
+    background: #27272E;
+  }
+ 
+  .scrollbar::-webkit-scrollbar-track {
+    background: rgba(36,46,82,.2)
+  }
+ }
```

```suggestion
 }
 
```
please enter the row